tests/unit/commands/__init__.py

```diff
@@ -49,7 +50,7 @@ def get_sample_key(name):
 
 
 def mock_check_output(command, *args, **kwargs):
-    if command[0].endswith("unsquashfs"):
+    if command[0].endswith("unsquashfs") or command[0].endswith("xdelta3"):
```

Comment:
This "generalized" mock_check_output is quite opinionated/coupled wrt the tests and fairly complex...

What do you think about trying to remove mock_check_output going forward by refactoring users of check_output to instead utilize a one-liner wrapper function that can be easily mocked as necessary? e.g.  `extract_squashfs(...)`, `xdelta_<op>(...)`, `snap_<op>(...)`?

Reply:
Yeah, this was part of an initiative I started but the prior team thought was too onerous for simple commands.